doc/release-notes-16528.md

```diff
@@ -0,0 +1,108 @@
+the wallet. Since descriptors are deterministic and easily enumerable, users will know exactly
+what scripts the wallet will consider to belong to it. Additionally the implementation of `IsMine`
+in Descriptor Wallets is far simpler than for Legacy Wallets. Notably, in Legacy Wallets, `IsMine`
+allowed for users to take on type of address (e.g. P2PKH), mutate it into another address type
+(e.g. P2WPKH), and the wallet would still detect outputs sending to the new address type
+even without that address being requested from the wallet. Descriptor Wallets does not
+allow for this and will only watch for the addresses that were explicitly requested from the wallet.
+
+These changes to `IsMine` will make it easier to reason about what scripts the wallet will
+actually be watching for in outputs. However for the vast majority of users, this change is
+largely transparent and will not have noticeable effect.
+
+#### Imports and Exports
+
+Descriptor Wallets handles imports differently from Legacy Wallets. In Legacy Wallets, imports
+are treated separately from the keys generated by the wallet. This separation is part of the
+watchonly behavior for Legacy Wallets and is also confusing, especially in combination with
+the `IsMine` behavior. With Descriptor Wallets, only complete descriptors can be imported. These
+descriptors are then just added to the wallet as if it were a descriptor generated by the wallet
+itself. To avoid issues with existing import and export RPCs, these have been disabled. To import
+into a Descriptor Wallet, a new `importdescriptors` RPC has been added that uses a syntax similar
+to that of `importmulti`.
+
+Because Descriptor Wallets are storing different records and data from Legacy Wallets, existing
+export RPCs are disabled for Descriptor Wallets. New export RPCs for Descriptor Wallets have
+not yet been added.
+
+The following RPCs have been disabled for Descriptor Wallets:
+
+* importprivkey
+* importpubkey
+* importaddress
+* importwallet
+* dumpprivkey
+* dumpwallet
+* importmulti
+* addmultisigaddress
+* sethdseed
+
+#### Watchonly Wallets
+
+In Legacy Wallets, a wallet could contain both private keys and scripts that were being watched.
+Those watched scripts would not contribute to your normal balance. In order to see the watchonly
+balance and to use watchonly things in transactions, an `include_watchonly` option as added
+to many RPCs that would allow users to do that. However this is confusing and easy to mess up.
+
+Descriptor Wallets move to a split watchonly model. Instead an entire wallet is considered to be
+watchonly depending on whether it was created with private keys disabled. This eliminates the need
+to distinguish between things that are watchonly and things that are not within a wallet itself.
+
+This change does have a caveat. If a Descriptor Wallet with private keys *enabled* and has
+both single key (e.g. wpkh(...)) and multiple key descriptors (e.g. multi(...) with only one private
```

Comment:
Is this inherent to mixed-use descriptor wallets?

For example, if you'd have a DW with private keys enabled and a `multi()` descriptor with only one private key, then the wallet would always fail to sign transactions, I expect?

Reply:
I've added a paragraph discussing that case